sound/soc/sof/sof-priv.h

```diff
@@ -40,7 +40,8 @@
 #define SOF_DBG_DUMP_MBOX		BIT(1)
 #define SOF_DBG_DUMP_TEXT		BIT(2)
 #define SOF_DBG_DUMP_PCI		BIT(3)
-#define SOF_DBG_DUMP_OPTIONAL		BIT(4) /* only dump if SOF_DBG_PRINT_ALL_DUMPS is set */
+/* only dump if SOF_DBG_PRINT_ALL_DUMPS is with KERN_DEBUG log level */
```

Comment:
```suggestion
/* Output this dump (at the DEBUG level) only when SOF_DBG_PRINT_ALL_DUMPS is set */
```
What is still not obvious when you look at just this .h file is which bits are global and which bits are per-dump. That shouldn't require finding and reading the code where they are used.
I'm guessing all the `SOF_DBG_DUMP_*` are per dump while all the others are global, correct? Also, are all the former direct module parameters? The comment should  mention which module and which parameter, perfect for `git grep`.
`/* debug flags */` adds absolutely zero information, should be replaced with something at least one-line long or removed. Comments should never just repeat  what's already obvious from the code.

Reply:
@marc-hb, The suggested comment look better, thanks. I will also add comment to clarify where these bits are supposed be.
Note: from the `SOF_DBG_DUMP_*` only the `SOF_DBG_DUMP_REGS` and `SOF_DBG_DUMP_OPTIONAL` is used, I'm tempted to drop the rest. In the past _none_ of these flags were used.